pkgs/applications/networking/bee/bee-clef.nix

```diff
@@ -0,0 +1,71 @@
+{ version ? "release", stdenv, lib, substituteAll
+, fetchFromGitHub, go-ethereum }:
+
+let
+
+  versionSpec = {
+    release = rec {
+      name = "bee-clef";
+      version = "0.4.4";
+      rev = "refs/tags/v${version}";
+      sha256 = "18fab0nxxsglrdykhb02m6vd7pgd6z4pqgfp5nmpxjcz7s8rfnml";
+    };
+  }.${version};
+
+in
+
+stdenv.mkDerivation {
+  pname = versionSpec.name;
+  version = versionSpec.version;
+
+  src = fetchFromGitHub {
+    owner = "ethersphere";
+    repo = "bee-clef";
+    rev = versionSpec.rev;
+    sha256 = versionSpec.sha256;
+  };
+
+  buildInputs = [ go-ethereum ];
+
+  patches = [
+    (substituteAll {
+      src = ./bee-clef.patch;
+      clef = "${go-ethereum}/bin/clef";
+      inherit outputDir;
+
+      # this fails, but i also need to somehow inject paths that are referring to files in $out 
+      #??? inherit out;
+    })
+  ];
+
+  buildPhase = " "; # we don't want the Makefile to be run
+
+  # this intentionally reproduces the same directory structure as it
+  # is in the upstream repo, but only copies the files that are
+  # needed.
+  installPhase = ''
+    mkdir -p $out/lib/systemd/system
+    cp packaging/clef-service $out
```

Comment:
Should go into `$out/bin` probably?

Reply:
or even into `$out/share/bee-clef/`? reasoning: i don't think we want this helper script in the path of the users who installed the package.

hrm, or maybe into `bin/` indeed, but then with a proper name: `bee-clef-service`. it might make sense to be able to start the service by hand in the foreground.

Reply:
on a second thought, i'll put this into share/bee-clef. it's not generally useful in the path.